### PR TITLE
Add size option to show more tasks without scrolling

### LINKED ISSUE #11: Show more tasks
Maybe I'm weird, but many of my projects have more than eight scripts. I know I can arrow down through them all, but it would be nice if `ntl` showed all of the scripts.

screen shot 2017-02-23 at 9 49 22 am

### PR DESCRIPTION
**Pull request by mischah**
Closes #11 

Still WIP because I have not added a test yet.
I tried hard to get this done. But I have no idea how to test this 😞 

I digged through the tests of Inquirer as well, but it didn’t enlighted me.

### CODE REVIEW

**Comment by mischah** [comment]
I asked Simon in the [Inquirer Gitter room](https://gitter.im/SBoudrias/Inquirer.js). Maybe he has an idea.

**Comment by mischah** [comment]
Mhmm. Travis fails on Node 0.12 because `meow` as dependency from `xo` uses `Object.assign()` 🤔

**Comment by mischah** [comment]
That’s the reason: https://github.com/npm/hosted-git-info/issues/31 😢

File: readme.md [hunk 1]
```diff
@@ -51,6 +52,7 @@ Options:
   -h --help      Shows this help message
   -a --all       Includes pre and post scripts on the list
   -m --multiple  Allows a selection of multiple tasks to run at once
+	-s --size      Sets the amount of tasks to show in the list without scrolling
```
Comment (ruyadorno) [inline readme.md]: oops, this has a tab instead of spaces here

Comment (mischah) [reply readme.md]: Oppsi.. Fixed that.

**Comment by ruyadorno** [comment]
I guess the sensible thing to do here is to just merge this as-is with no tests, seeing that it will probably be a temporary solution before the v2 refactor it's not too bad

as long as this is not breaking any of the existing tests, I'm fine with it 👍

**Comment by ruyadorno** [comment]
thank you so much @mischah 🎉🎉🎉

**Comment by zeke** [comment]
Nice!

**Pull request merged**